package.json

```diff
@@ -22,7 +22,7 @@
     "int-test-ui": "cypress open",
     "record-build-info": "node ./bin/record-build-info",
     "security_audit": "npx audit-ci --config audit-ci.json",
-    "lint": "eslint . --cache --max-warnings 0",
+    "lint": "eslint . --cache --max-warnings 0 && stylelint **/*.scss",
```

Comment:
Will Stylelint know to exclude the files in `node_modules`?

Reply:
Apparently, yes - according to [this](https://stylelint.io/user-guide/configure#ignorefiles):

> stylelint ignores the node_modules directory by default. However, this is overridden if ignoreFiles is set.